### PR TITLE
Productionize llo transmitter

### PR DESCRIPTION
**Pull request by samsondav**
Productionize LLO transmitter.

This mostly copies over similar patterns and structure from the battle-tested mercury transmitter, with a few modifications to generalize it to LLO.

### CODE REVIEW

File: core/cmd/forwarders_commands_test.go [hunk 1]
```diff
@@ -22,7 +22,7 @@ func TestEVMForwarderPresenter_RenderTable(t *testing.T) {
 	t.Parallel()
 
 	var (
-		id         = "1"
+		id         = "ID:"
```
Comment (samsondav) [inline core/cmd/forwarders_commands_test.go:25]: This was needed to fix a flake

File: core/services/llo/mercurytransmitter/server.go [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+	var payload []byte
+	var err error
+
+	switch t.Report.Info.ReportFormat {
+	case llotypes.ReportFormatJSON:
+		// TODO: exactly how to handle JSON here?
+		// https://smartcontract-it.atlassian.net/browse/MERC-3659
+		fallthrough
+	case llotypes.ReportFormatEVMPremiumLegacy:
+		payload, err = evm.ReportCodecPremiumLegacy{}.Pack(t.ConfigDigest, t.SeqNr, t.Report.Report, t.Sigs)
```
Comment (samsondav) [inline core/services/llo/mercurytransmitter/server.go:225]: we end up packing each time for each server here, is that correct?

File: core/services/ocr2/plugins/llo/config/config.go [hunk 3]
```diff
@@ -41,31 +40,40 @@ type PluginConfig struct {
...
+
 func (p PluginConfig) Validate() (merr error) {
 	if p.DonID == 0 {
 		merr = errors.Join(merr, errors.New("llo: DonID must be specified and not zero"))
 	}
 
-	if p.RawServerURL == "" {
-		merr = errors.Join(merr, errors.New("llo: ServerURL must be specified"))
+	if len(p.Servers) == 0 {
+		merr = errors.Join(merr, errors.New("llo: At least one Mercury server must be specified"))
```
Comment (msuchacz-cll) [inline core/services/ocr2/plugins/llo/config/config.go:68]: Should it be "Mercury server"? this transmitter is purely for Data Engine?

Comment (samsondav) [reply core/services/ocr2/plugins/llo/config/config.go:68]: Well, in this case the nomenclature is shifting a bit. I’m still referring to the server as “mercury”,  this is specific to transmitting to the server.

The plugin itself does not reference it

**Review by brunotm — DISMISSED** [review]

**Review by msuchacz-cll — DISMISSED** [review]

**Review by akuzni2 — DISMISSED** [review]

File: core/services/llo/mercurytransmitter/server.go [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+		if res.Error == "" {
+			s.transmitSuccessCount.Inc()
+			s.lggr.Debugw("Transmit report success", "transmission", t, "response", res)
+		} else {
+			// We don't need to retry here because the mercury server
+			// has confirmed it received the report. We only need to retry
+			// on networking/unknown errors
+			switch res.Code {
+			case DuplicateReport:
+				s.transmitSuccessCount.Inc()
```
Comment (akuzni2) [inline core/services/llo/mercurytransmitter/server.go:198]: are these referencing the `llo_mercury_transmit_` prefixed metrics? could just be the github review tool but it's pulling up that it's point to `core/services/relay/evm/mercury/transmitter.go` when it should be `core/services/llo/mercurytransmitter/transmitter.go`

**Review by brunotm — DISMISSED** [review]

File: core/services/llo/mercurytransmitter/server.go (hunk 4 above)
Comment (samsondav) [reply core/services/llo/mercurytransmitter/server.go:198]: Yeah they point to the `llo_mercury_transmit_*` metrics

**Review by brunotm — APPROVED** [review]

**Review by msuchacz-cll — APPROVED** [review]

**Pull request merged**